Description: Add Keywords entry to .desktop file
 lintian warning desktop-entry-lacks-keywords-entry
 usr/share/applications/htop.desktop
Author: Daniel Lange <dl.ml1@usrlocal.de>
---
--- a/htop.desktop
+++ b/htop.desktop
@@ -60,3 +60,4 @@ GenericName[sv]=Processvisning
 GenericName[tr]=Süreç Görüntüleyici
 GenericName[uk]=Перегляд процесів
 GenericName[zh_TW]=行程檢視器
+Keywords=system;process;task
